src/main/webapp/WEB-INF/views/admin/project/list-js.jsp

```diff
@@ -957,7 +957,9 @@
 					alertify.error('<spring:message code="msg.common.valid2" />', 0);
 				}
 			});
-		}
+		},toolTipDoubleclick: function () {
+            return 'title="' + "Double click" + '"';
+        }
```

Comment:
Indentation of 961~962 line should be fixed

Reply:
thank you for code review!! 😊👍
i'll fix it!!